distbench_engine.cc

```diff
@@ -930,8 +1006,19 @@ void DistBenchEngine::RunAction(ActionState* action_state) {
         [this](std::shared_ptr<ActionIterationState> iteration_state) {
           RunRpcActionIteration(iteration_state);
         };
+  } else if (action.proto.has_activity_name()) {
+    // TODO:
+    // Currently we have only one activity. In future,
+    // make a map of activity_name to function.
+    action_state->iteration_function =
+        [this,
+         action_state](std::shared_ptr<ActionIterationState> iteration_state) {
+          WasteCpuCycles(2500);
```

Comment:
This looks like it says "waste 2500 cpu cycles" maybe we should drop the "cycles"